composer.json

```diff
@@ -38,6 +38,7 @@
         "symfony/validator": "^2.2",
         "symfony/form": "~2.1",
         "symfony/filesystem": "^2.1",
+        "symfony/expression-language": "^2.6|^3.0",
```

Comment:
The `composer.lock` file is not committed to git. So you need to run `composer update` instead.